queries/latex/highlights.scm

```diff
@@ -137,26 +140,6 @@
   name: (curly_group_text
     (_) @markup.link))
 
-; Formatting
-(text_mode
-  content: (curly_group
-    (_) @none @spell))
-
-(math_environment
-  (begin
-    command: _ @markup.math
-    name: (curly_group_text
-      (_) @markup.math)))
-
-(math_environment
-  (_) @markup.math)
-
-(math_environment
```

Comment:
These were deliberate to mimic VimTeX syntax highlighting (which is the high watermark).

I'm not averse on principle, but that looks like a bigger change that should be discussed.

Reply:
But weren't `(math_environment (begin/end ...` straight up redundant, when
```
(math_environment
  (_) @markup.math)
```
(which is still there, just moved) gives `@markup.math` to all children of `(math_environment)`?

Reply:
Also, `(text_mode content:...` was just moved/combined.